package.json

```
@@ -150,6 +150,6 @@
     ]
   },
   "dependencies": {
-    "contentful-management": "^7.23.0"
+    "contentful-management": "7.27.0"
```

Why did you pin the dependency?

```suggestion
    "contentful-management": "^7.27.0"
```

---

it was pinned in master when it was under `devDependencies`: 
https://github.com/contentful/ui-extensions-sdk/blob/1b474e136ff75cda86f20036e00a31f51bef29da/package.json#L58

---

It's fine to pin dev dependencies as people installing app-sdk do not install those. But regular dependencies should not be pinned as we fix the cma.js our users can use which will cause issues.

---

Makes sense 👍